config/zones/FI.yaml

```diff
@@ -53,7 +53,7 @@ emissionFactors:
     gas:
       datetime: '2021-01-01'
       source: EU-ETS, ENTSO-E 2021
-      value: 684.2607818
```

Comment:
Any reason for why the change here is so massive?

Reply:
there was also probably one power plant with a suspiciously high EF which we have now removed from the set